src/Constraints/CollectionValidator.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tuf\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Constraints\Collection;
+use Symfony\Component\Validator\Constraints\CollectionValidator as SymfonyCollectionValidator;
+
+class CollectionValidator extends SymfonyCollectionValidator
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        if (!empty($constraint->unsupportedFields)) {
```

Comment:
I'm not sure we need this enclosing `if` statement, since $constraint->unsupportedFields defaults to an empty array.

Reply:
Removing